tests/unit/lib/testChangelog.test.js

```diff
@@ -211,5 +213,47 @@ describe("ChangelogLib lib", () => {
           "- test1 [#1](http://gitlab.example.com/my-group/my-project/merge_requests/1) ([admin](https://gitlab.example.com/admin))\n");
       });
     });
```

Comment:
Can we have a test case with fullChangeLogLink please?

Reply:
Added, but as mentioned in the description, I'm unable to test (or could you explain how to test?)